src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -948,14 +954,16 @@ protected function build_attendee_data( array $attendee, $context = 'default' )
 
 		// Sensible information should not be shown to everyone
 		if ( $has_manage_access ) {
-			$attendee_data = array_merge( $attendee_data, array(
-				'provider'        => $this->get_provider_slug( $provider ),
-				'order'           => $attendee_order_id,
-				'sku'             => $this->get_attendee_sku( $attendee_id, $attendee_order_id, $provider ),
-				'email'           => Tribe__Utils__Array::get( $attendee, 'holder_email', Tribe__Utils__Array::get( $attendee, 'purchaser_email', '' ) ),
-				'checked_in'      => $checked_in,
-				'checkin_details' => $checkin_details,
-			) );
+			$attendee_data = array_merge(
+				$attendee_data, [
```

Comment:
:no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 9b8d1162d955b6108f74f62074deab49</sub>

Reply:
fixed